conf/mailetcontainer.xml

```diff
@@ -1,149 +0,0 @@
-        <name>bcc</name>
-        <onMailetException>ignore</onMailetException>
-      </mailet>
-      <mailet match="All" class="RecipientRewriteTable">
-        <errorProcessor>rrt-error</errorProcessor>
-      </mailet>
-      <mailet match="RecipientIsLocal" class="Sieve"/>
-      <mailet match="RecipientIsLocal" class="AddDeliveredToHeader"/>
-      <mailet match="RecipientIsLocal" class="LocalDelivery"/>
-      <mailet match="HostIsLocal" class="ToProcessor">
-        <processor>local-address-error</processor>
-        <notice>550 - Requested action not taken: no such user here</notice>
-      </mailet>
-
-      <mailet match="relay-allowed" class="ToProcessor">
-        <processor>relay</processor>
-      </mailet>
-    </processor>
-
-    <processor state="relay" enableJmx="true">
-      <mailet match="All" class="RemoteDelivery">
-        <outgoingQueue>outgoing</outgoingQueue>
-        <delayTime>5000, 100000, 500000</delayTime>
-        <maxRetries>25</maxRetries>
-        <maxDnsProblemRetries>0</maxDnsProblemRetries>
-        <deliveryThreads>10</deliveryThreads>
-        <sendpartial>true</sendpartial>
-        <bounceProcessor>bounces</bounceProcessor>
-      </mailet>
-    </processor>
-
-    <processor state="local-address-error" enableJmx="true">
-      <mailet match="All" class="MetricsMailet">
-        <metricName>mailetContainerLocalAddressError</metricName>
-      </mailet>
-      <mailet match="All" class="Bounce">
-        <attachment>none</attachment>
-      </mailet>
-      <mailet match="All" class="ToRepository">
-        <repositoryPath>file://var/mail/address-error/</repositoryPath>
-      </mailet>
-    </processor>
-
-    <processor state="relay-denied" enableJmx="true">
-      <mailet match="All" class="MetricsMailet">
-        <metricName>mailetContainerRelayDenied</metricName>
-      </mailet>
-      <mailet match="All" class="Bounce">
-        <attachment>none</attachment>
-      </mailet>
-      <mailet match="All" class="ToRepository">
-        <repositoryPath>file://var/mail/relay-denied/</repositoryPath>
-        <notice>Warning: You are sending an e-mail to a remote server. You must be authenticated to
-          perform such an operation
-        </notice>
-      </mailet>
-    </processor>
-
-    <processor state="bounces" enableJmx="true">
-      <mailet match="All" class="MetricsMailet">
-        <metricName>bounces</metricName>
-      </mailet>
-      <mailet match="All" class="DSNBounce">
-        <passThrough>false</passThrough>
-      </mailet>
-    </processor>
-
-    <processor state="rrt-error" enableJmx="false">
-      <mailet match="All" class="ToRepository">
-        <repositoryPath>file://var/mail/rrt-error/</repositoryPath>
-        <passThrough>true</passThrough>
-      </mailet>
-      <mailet match="IsSenderInRRTLoop" class="Null"/>
-      <mailet match="All" class="Bounce"/>
-    </processor>
-
-    <processor state="do-nothing-mailet" enableJmx="false">
```

Comment:
You removed my custom mailets.

Reply:
Yes, you are right. I will cancel this PR and try a different approach.